### ISSUE TITLE
Restructure buildpack: Update composer groups in language family

### ISSUE THREAD

**Issue by sophiewigmore** [paketo-buildpacks/php#475]
## Context
Per [PHP RFC 0001](https://github.com/paketo-buildpacks/rfcs/blob/main/text/php/0001-restructure.md) we are restructuring the PHP language family of buildpacks to make them more modular, up-to-date, and as a result, maintainable.

## Issue
This issue is to update the language family order groupings to replace `php-composer` with `composer` and `composer-install`.

**This issue is blocked on https://github.com/paketo-buildpacks/php/issues/473.** Once completed, both `composer` and `composer-install` can be added to the buildpack.toml order groupings,

Please update the README accordingly.

## Acceptance
1. The order grouping in the PHP Language family buildpack looks like: 
```
...
  [[order.group]]
    id = "paketo-buildpacks/php-dist"
    version = ...

  [[order.group]]
    id = "paketo-buildpacks/composer"
    optional = true
    version = ...

  [[order.group]]
    id = "paketo-buildpacks/composer-install
    optional = true
    version = ...
...
```
2. I can build a composer app such as the [Paketo composer sample app](https://github.com/paketo-buildpacks/samples/tree/main/php/composer) with this version of the PHP language family.

**Comment by sophiewigmore** [comment]
Closing in favour of #524
